src/main/resources/feedbackQuestionMcqEditFormWeightFragment.html

```diff
@@ -0,0 +1,5 @@
+<div class="margin-bottom-7px mcqChoiceRow-${i}-${questionNumber}">
```

Comment:
If you are really going to use this class name in the end then rmb to put it in `Const`.